dev/com.ibm.ws.concurrent.java8/resources/com/ibm/ws/concurrent/resources/CWWKCMessages.nlsprops

```diff
@@ -0,0 +1,87 @@
+CWWKC1000.unrecognized.property.explanation=An unrecognized property is configured on a context service.
+CWWKC1000.unrecognized.property.useraction=Remove or correct the unrecognized property.
+
+# Moved from com.ibm.ws.context
+CWWKC1001.context.providers.unavailable=CWWKC1001E: The configured thread context {0} is unavailable for the configuration element {1}.
+CWWKC1001.context.providers.unavailable.explanation=The application server is unable to find provider(s) of the specified type(s) of thread context.
+CWWKC1001.context.providers.unavailable.useraction=Ensure the provider(s) of the requested thread context are enabled.
+
+# Moved from com.ibm.ws.context
+CWWKC1002.provider.cardinality.violation=CWWKC1002E: The configuration element {0} cannot have more than one sub-element of type {1}.
+CWWKC1002.provider.cardinality.violation.explanation=At most, one element of each type of context provider may be configured under a context service.
+CWWKC1002.provider.cardinality.violation.useraction=Remove duplicate context provider elements from the context service configuration.
+
+# Moved from com.ibm.ws.context
+CWWKC1020.baseContextRef.infinite=CWWKC1020E: The attribute baseContextRef of configuration element {0} causes an infinite chain of dependencies.
+CWWKC1020.baseContextRef.infinite.explanation=An infinite chain of dependencies exists because baseContextRef directly or indirectly points back to the same context service instance.
+CWWKC1020.baseContextRef.infinite.useraction=Update the context service configuration to break the infinite chain of dependencies.
+
+CWWKC1100.resource.unavailable=CWWKC1100E: The resource {0} is unavailable.
+CWWKC1100.resource.unavailable.explanation=A resource might become unavailable due to the server shutting down or because of a configuration change, either to itself or another service upon which it depends.
+CWWKC1100.resource.unavailable.useraction=Verify the configuration of the resource and any resources upon which it depends.
+
+CWWKC1110.task.canceled=CWWKC1110I: The task {0}, which was submitted to executor service {1}, is canceled.
+CWWKC1110.task.canceled.explanation=Execution of the task has been canceled.
+CWWKC1110.task.canceled.useraction=No action is required.
+
+CWWKC1101.task.failed=CWWKC1101E: The task {0}, which was submitted to executor service {1}, failed with the following error: {2}.
+CWWKC1101.task.failed.explanation=A task submitted to an executor has failed. Refer to the exception message for the cause.
+CWWKC1101.task.failed.useraction=Correct the cause of the failure and resubmit the task.
+
+CWWKC1102.listener.failed=CWWKC1102E: The managed task listener for task {0}, which was submitted to executor service {1}, failed with the following error: {2}.
+CWWKC1102.listener.failed.explanation=A managed task listener operation has failed. Refer to the exception message for the cause.
```

Comment:
```suggestion
CWWKC1102.listener.failed.explanation=A managed task listener operation fails. Refer to the exception message for the cause.
```

Reply:
"operation failed" ("failed", not "has failed" or "fails")